gemd/entity/value/base_value.py

```diff
@@ -12,3 +15,16 @@ class BaseValue(DictSerializable):
     """
 
     typ = "value"
+
+    @abstractmethod
+    def to_bounds(self) -> BaseBounds:
```

Comment:
Is there a use case for this method outside of `contains`?  Should this method be "protected"?

Reply:
The obvious use case would be building the union of two bounds.  It's not obvious that this is the cleanest implementation, but a method call like
```
full_range = value1.to_bounds.union(value2.to_bounds)
```
might make sense (unattractive sig, yes).  In particular, 
```
full_range = value1.union(value2)
```
probably _wouldn't_ make sense.  Another viable option would be
```
full_range = BaseBounds(value1).union(value)
```
but the inheritance gets a little confused (i.e., I called a static method on CategoricalBounds but it returned a RealBounds).  

This last scenario would allow changing this to a protected method.

Reply:
I'd be inclined to make it protected until `union` exists.  Its easy to make a protected method public than to make a public method protected.